Uplift/Views/Onboarding/CreateProfileView.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  CreateProfileView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import PhotosUI
+import SwiftUI
+
+struct CreateProfileView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @State private var didCheckData: Bool = false
+    @State private var didCheckLocation: Bool = false
+    @State private var didCheckTerms: Bool = false
+    @State private var didShowImagePicker = false
+    @State private var profileImage: UIImage?
+    @State private var profileItem: PhotosPickerItem?
+
+    // MARK: - UI
+
+    var body: some View {
+        ZStack {
+            Color.white
+                .ignoresSafeArea()
+
+            VStack {
+                HStack {
+                    Text("Complete your profile.")
+                        .font(Constants.Fonts.h1)
+                        .padding(.leading, 16)
+
+                    Spacer()
+                }
+
+                DividerLine()
+                    .upliftShadow(Constants.Shadows.smallLight)
+                    .padding(.bottom, 46)
+
+                cameraPlaceholderButton
+
+                Text(mainViewModel.name)
+                    .font(Constants.Fonts.h1)
+
+                CheckBoxView(
+                    didCheckData: $didCheckData,
+                    didCheckLocation: $didCheckLocation,
+                    didCheckTerms: $didCheckTerms
+                )
+                .padding(.top, 48)
+
+                if allChecked {
+                    Spacer(minLength: 135)
+
+                    readyGreeting
+                    getStartedButton
+
+                } else {
+                    Spacer(minLength: 200)
+
+                    nextLabel
+                }
+
+                Spacer()
+            }
+            .padding(.vertical, 40)
+            .photosPicker(
+                isPresented: $didShowImagePicker,
+                selection: $profileItem,
+                matching: .images,
+                photoLibrary: .shared()
+            )
+            .onChange(of: profileItem) { newItem in
+                Task {
+                    if let newItem,
+                       let data = try? await newItem.loadTransferable(type: Data.self),
+                       let image = UIImage(data: data) {
+                        DispatchQueue.main.async {
+                            self.profileImage = image
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private var allChecked: Bool {
+        didCheckTerms && didCheckData && didCheckLocation
+    }
```

Comment:
Can this be moved to the bottom of the file under a MARK: - Helpers (or something similar) since it's not a UI component?